src/modules/users/edit-user/edit-user.html

```diff
@@ -42,7 +42,7 @@
             <div class="form-group">
                 <label class="col-sm-2 col-md-1 control-label cp-padding" for="password">New Password</label>
                 <div class="col-sm-8 col-md-6 inner-addon">
-                    <input type="{{ editUserCntrl.typePassword ? 'text' : 'password' }}" data-id="password" name="password" ng-model="editUserCntrl.user.password" minlength="8" />
+                    <input type="{{ editUserCntrl.typePassword ? 'text' : 'password' }}" data-id="password" name="password" ng-model="editUserCntrl.user.password" minlength="9" ng-maxlength="128"/>
```

Comment:
if you have used ng-maxlength then why you have not used ng-minlength?

Reply:
@cloudbehl ng-minlength/ng-maxlength don't apply default html validation messages which  applied when minlength/maxlength are used. Hence, it make sense to use minlength. Why I have not used maxlength is  because using maxlength stops the user from entering any more characters(once maxlength limit cross) without showing any validation message which do not provide a better user experience. Hence, I need to handle that scenario using ng-maxlength.

For login page, I have used ng for both because of security implications on login page which don't allow us to show the max or min length required. Please refer Ju's comment https://bugzilla.redhat.com/show_bug.cgi?id=1594994#c12 for more detail.